openedx/features/content_type_gating/tests/test_access.py

```diff
@@ -59,7 +57,7 @@
 
 
 @patch("crum.get_current_request")
-def _get_content_from_fragment(block, user_id, course, request_factory, mock_get_current_request):
+def _get_content_from_fragment(_store, block, user_id, course, request_factory, mock_get_current_request):
```

Comment:
It doesn't look like _store is ever used here

Reply:
Yeah, that's why I named it with an underscore. I added it to this function because in the tests, this method and `_get_content_from_lms_index` are both used with the same arguments (see `_assert_block_is_gated`) - so they need the same arguments signature.